app/scripts/views/settings/emails.js

```diff
@@ -27,6 +28,7 @@ define(function (require, exports, module) {
 
   var View = FormView.extend({
     template: Template,
+    gatedTemplate: Template,
```

Comment:
Could this be passed into the mixin's options? Or perhaps, could the mixin pull that info from `this.template` in its `initialize` function without relying on an additional property?